src/project/src/main/scala/io/giovannini/Main.scala

```diff
@@ -2,7 +2,7 @@ package io.giovannini
 
 object Main extends App {
 
-  val result = EvenFibonnacci.sumFilterFibonacciUnder(4000000, _ % 2 < 1)
+  val result = EvenFibonnacci.sumFilterFibonacciUnder(4000000)
```

Comment:
🙅 [error] not enough arguments for method sumFilterFibonacciUnder: (n: Int, condition: Int => Boolean)Int.

Reply:
@guyonvarch Met un croissant plutôt que ce bonhomme. Le mec n'a pas compilé son code quoi ^^